NuGet.config

```diff
@@ -3,7 +3,6 @@
   <packageSources>
     <clear />
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
-    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
```

Comment:
Is this just incidental, or is it possible due to some other change?

Reply:
Intentional. As a part of lineups, we snap Internal.AspNet.Sdk and API check PackageReferences to the version bundled in KoreBuild.

cref https://github.com/aspnet/BuildTools/blob/03a29fe5f7fb1eace6ae85351bf708ca8b851144/modules/BundledPackages/module.targets#L4